Source/WebKit/Shared/WebHitTestResultData.cpp

```diff
@@ -114,6 +126,7 @@ WebHitTestResultData::WebHitTestResultData(const HitTestResult& hitTestResult, c
 
     elementType = elementTypeFromHitTestResult(hitTestResult);
     linkLocalDataMIMEType = linkLocalDataMIMETypeFromHitTestResult(hitTestResult);
+    linkLocalDataSuggestedFilename = linkLocalDataSuggestedFilenameFromHitTestResult(hitTestResult);
```

Comment:
I think these should all be moved to the initializers above and out of the constructor body.

Reply:
Done. I also refactored the constructors to remove duplication of that long initializer list.